misc/kind/cluster.yaml

```diff
@@ -153,6 +153,11 @@ nodes:
       - containerPort: 32063
         hostPort: 32063
 
+  - role: worker
```

Comment:
I think you need to rebase . Now that disk replicas are the default, cluster.yaml I think has changed.